tests/api/test_dnf_rpm.py

```diff
@@ -6,14 +6,15 @@
 
 import dnf
 
+import tempfile
```

Comment:
This looks like a leftover from debugging, it's not needed?

Reply:
I am sorry, I tested multiple way.